modules/cnv-live-migration-limits-reftable.adoc

```diff
@@ -0,0 +1,36 @@
+
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-live-migration-limits.adoc
+
+[id="cnv-live-migration-limits-ref_{context}"]
+= Cluster-wide live migration limits
+
+.Migration limits
+|===
+|Parameter |Description |Default
+
+|`parallelMigrationsPerCluster`
+|Number of migrations running in parallel in the cluster. 
+|5
+
+|`parallelOutboundMigrationsPerNode`
+|Maximum number of outbound migrations per node.
+|2
+
+|`bandwidthPerMigration`
+|Bandwidth limit of each migration, in MiB/s.
+|64Mi
+
+|`completionTimeoutPerGiB`
+|The migration will be aborted if it has not completed in this time, in seconds 
+per GiB of memory. For example, a VMI with 6GiB memory will timeout if it has 
+not completed migration in 4800 seconds.
```

Comment:
In case of a block migration, the size of the disks that are being migrated will be included in the calculation.

Reply:
Now reads:
"The migration will be aborted if it has not completed in this time, in seconds
per GiB of memory. For example, a VMI with 6GiB memory will timeout if it has
not completed migration in 4800 seconds. If the `Migration Method` is
`BlockMigration`, the size of the migrating disks is included in the calculation."